### PR TITLE
Added team member profiles

### PR DESCRIPTION
**Pull request by pranavkarthik10**
# Overview
Fixes issue #608. Added profile image with social profiles for **Akshay Shirmali** and **Chung Wei Tat**. Please let me know if their title or section needs to be changed. 

# Preview
View changes at https://pranavkarthik10.github.io/fossasia.org/team/
Screen Shot 2019-12-24 at 8 08 04 PM
Screen Shot 2019-12-24 at 8 08 11 PM

### CODE REVIEW

**Comment by liveHarshit** [comment]
On what basis are you adding team member. All GCI mentors are not part of the team. @mariobehling Am I right?

**Comment by pranavkarthik10** [comment]
@liveHarshit I don't believe they are GCI mentors. I was looking through their LinkedIn Profiles to find what role they have in FOSSASIA.

**Comment by liveHarshit** [comment]
> @liveHarshit I don't believe they are GCI mentors. I was looking through their LinkedIn Profiles to find what role they have in FOSSASIA.

add Chung Wei Tat only. Akshay is a new contributor. Try to add GSoC students only.

**Comment by pranavkarthik10** [comment]
@liveHarshit Updated with new commit!

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,2 @@
 id_rsa
-.DS_Store
\ No newline at end of file
+.DS_Store
```
Comment (liveHarshit) [inline .gitignore]: Unnecessary changes. Revert it. Reset looks good :+1:

**Review by liveHarshit — CHANGES_REQUESTED** [review]

File: team/index.html [hunk 2]
```diff
@@ -1560,7 +1560,21 @@ <h1>Enablement Team</h1>
...
 					</div>
 				</div>
-
+				<div class="col-md-3 col-sm-6 speaker-column">
+				<div class="speaker">
+					<div class="image-holder">
+						<img class="background-image" alt="Chung Wei Tat" src="../img/team/ChungWeiTat.jpg">
+						<div class="hover-state text-center preserve3d">
+							<div class="social-links vertical-align">
+								<a href="https://twitter.com/cweitat"><i class="icon social_twitter"></i></a>
```
Comment (liveHarshit) [inline team/index.html]: Open page in a new tab. With target = blank

File: .gitignore (hunk 1 above)
Comment (pranavkarthik10) [reply .gitignore]: Fixed!

File: team/index.html (hunk 2 above)
Comment (pranavkarthik10) [reply team/index.html]: Now updated to open in new tab.

**Review by liveHarshit — APPROVED** [review]
